exercises/concept/monster-attack/.meta/design.md

```diff
@@ -0,0 +1,29 @@
+# Design
+
+## Learning objectives
+
+- Know that calling a function without all its arguments returns a new function.
+- Understand the arrows notation `->` in type annotations.
+- Know how convenient is partial applications in pipes `|>`.
+- Understand how partial application and pipes encourage placement of the main data structure as last argument of functions.
+
+## Out of scope
+
+- Composition of partial functions is out of scope, left for another concept.
+
+## Concepts
+
+- partial-application-composition
+
+## Prerequisites
+
+- basics-2
+
+## Analyzer
+
+This exercise could benefit from the following rules added to the the [analyzer][analyzer]:
+
+- Verify that the pipeline operator is used in `attack`
```

Comment:
Update for attack1/2/3
The analyzer can't check the presence of operators yet, I need to add that (but it's simple, I did it for `let` recently)